management/server/account.go

```diff
@@ -230,6 +231,14 @@ type Account struct {
 	RulesG []Rule           `json:"-" gorm:"-"`
 }
 
+// AccountUsageStats represents the current usage statistics for an account
+type AccountUsageStats struct {
+	ActiveUsers int64 `json:"active_users"`
+	TotalUsers  int64 `json:"total_users"`
```

Comment:
We need to have a better breakdown and separately count service users, regular users and users provisioned via integrations.

Reply:
I've excluded service users from the total user count: 396fae516126a3ae16cfdba66dd43fa96bc45624
Active users only take active peers into account, which service users do not have, so this is fine.